data-tool/flows/batch_delete_flow.py

```diff
@@ -0,0 +1,444 @@
+
+            query_plans = [
+                # based on transaction_id
+                {
+                    'source': 'addresses_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'aliases_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'amalgamating_businesses_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'amalgamations_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'documents_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'jurisdictions_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'offices_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'parties_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'party_roles_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'resolutions_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'share_classes_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                {
+                    'source': 'share_series_version',
+                    'params': {'transaction_id': transaction_ids},
+                },
+                # based on others
+                {
+                    'source': 'batch_processing',
+                    'columns': ['id', 'batch_id'],
+                    'params': {'business_id': business_ids},
+                    'targets': ['batch_processing', 'batches'],
+                },
+                {
+                    'source': 'colin_event_ids',
+                    'columns': ['colin_event_id'],
+                    'params': {'filing_id': filing_ids},
+                },
+                # there're some Comment records saved by legal-api directly instead of filer
+                # should ignore them (assume that it won't be populated with migrated data)
+                {
+                    'source': 'comments',
+                    'params': {'filing_id': filing_ids},
+                },
+                {
+                    'source': 'consent_continuation_outs',
+                    'params': {'filing_id': filing_ids},
+                },
```

Comment:
I think we need to delete the entries where the comment data is directly linked via the business_id too